prow/cmd/deck/main.go

```diff
@@ -1274,6 +1287,21 @@ func canTriggerJob(user string, pj prowapi.ProwJob, cfg *prowapi.RerunAuthConfig
 	}
 	if cli == nil {
 		logrus.Warning("No GitHub token was provided, so we cannot retrieve GitHub teams")
+		return false, nil
+	}
+
+	// If the job is a presubmit and has an associated PR, do the same checks as for /test
+	if pj.Spec.Type == prowapi.PresubmitJob && pj.Spec.Refs != nil && len(pj.Spec.Refs.Pulls) > 0 {
+		if pluginAgent == nil {
```

Comment:
you could add `&& pluginAgent != nil` to the above and get the same effect, right?

Reply:
Yes... I'll also change this in a followup since I want to get this merged today

Reply:
^ Fixes made in #13865